python/ray/experimental/gpu_object_manager/gpu_object_manager.py

```diff
@@ -58,6 +58,10 @@ def __init__(self):
         # This dictionary is hosted on the "driver" process of the actors that
         # store and send/receive GPU objects.
         self.managed_gpu_object_metadata: Dict[str, GPUObjectMeta] = {}
+
+        # Store the pending receive GPU object metadata.
+        self.pending_gpu_object_metadata: Dict[str, GPUObjectMeta] = {}
```

Comment:
Why do we need a separate map here instead of putting into self.managed_gpu_object_metadata?